snapper.py

```diff
@@ -495,34 +495,36 @@ def run_scrub():
 
         return None
 
-    if scrub_delayed_run and scrub_delayed_run > 0:
+    # this block only runs for users who have scrub_delayed_run nonzero (enabled)
```

Comment:
This check should prevent any "delayed scrub" features and related message from running for users who have the configuration set to 0